### PR TITLE
IlliniHackers: Queue Dev Night Recap

### PR DESCRIPTION
**Pull request by apache8080**
I have an initial draft of the recap post. I still need to add links and images.

### CODE REVIEW

**Comment by redsn0w422** [comment]
We should also link to the Queue and also the PR's made

**Comment by apache8080** [comment]
Yeah, the puppeteer thing was an accident, and I will add the links and images later today.

File: _posts/2018-04-24-illinihackers-queue-dev-night-recap.markdown [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title:  "The Return of the HackNight"
+date:   2018-04-24 12:00:00
+categories: recap
+tags: IlliniHackers, HackNight
+image: /assets/article_images/rust_cover.jpg
+author: Rishi Desai
+---
+```
+
+For the HackIllinois team, open source never stops. This Tuesday we brought back our open source HackNights. HackNights are an opportunity for students on the UIUC campus to work on open source projects with peers and professors. For our first HackNight, we invited the Illinois Queue team to teach and work with students to improve the Queue. The Queue is a campus wide open source projects used by classes to keep track of students waiting in line for office hours. Nathan Walters, Genna Helsel, and Professor Wade Fagen Ulmschneider from the Queue development team started off the night by giving a short tutorial on how to get started, and then the 20 students who came began contributing to the project. After 2 productive hours of work, there were 5 Pull Requests made and 4 new Issues logged for the project. This was the first of many HackNights and future open source contributions. Stay tuned for more updates and keep on contributing.
```
Comment (rohinb2) [inline _posts/2018-04-24-illinihackers-queue-dev-night-recap.markdown]: For the HackIllinois team, open source never stops. This past Tuesday, we continued our work in open source through an open source HackNight. HackNights are an opportunity for student developers at UIUC to work on open source projects with peers, professors, and lead engineers. For our first HackNight, we invited the developers of the [Illinois Queue](https://github.com/illinois/queue) team to teach and work with students to improve the Queue - one of Illinois's latest and greatest open source projects. The Queue is a campus wide tool used by classes to keep track of students waiting in line for office hours. 

Nathan Walters, Genna Helsel, and Professor Wade Fagen Ulmschneider from the Queue development team started off the night by giving a short tutorial on how to get started, and then it was off to the races as the students at the HackNight started claiming issues and making PRs. After just 2 hours of work, there were 5 Pull Requests made and 4 new Issues logged for the project! 

Our goal with our HackNights is to encourage continuous open source contribution throughout the year - especially on projects that are relevant to the community. As development on the queue continues, we hope to encourage contribution to more projects in the open source community (up next, autograder!). Of course, this was the first of many HackNights and just a start for the larger community of Illinois Open Source. Stay tuned for more updates and, as always, keep on contributing!

**Comment by apache8080** [comment]
I don't know if an inline image is necessary.

**Comment by apache8080** [comment]
@rohinb2 @redsn0w422 This should be the final edit. The only thing that doesn't work is the author thing; it still says it is from the HackIllinois Team.

File: _posts/2018-04-24-illinihackers-queue-dev-night-recap.markdown [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 ---
-```
+
+
 
 For the HackIllinois team, open source never stops. This past Tuesday, we continued our work in open source through an open source HackNight. HackNights are an opportunity for student developers at UIUC to work on open source projects with peers, professors, and lead engineers. For our first HackNight, we invited the developers of the [Illinois Queue](https://github.com/illinois/queue) team to teach and work with students to improve the Queue - one of Illinois's latest open source projects. The Queue is a campus wide tool used by classes to keep track of students waiting in line for office hours.
 
+## The Night
+
 Nathan Walters, Genna Helsel, and Professor Wade Fagen Ulmschneider from the Queue development team started off the night by giving a short tutorial on how to get started, and then it was off to the races as the students at the HackNight started claiming issues and making PRs. After just 2 hours of work, there were [5 Pull Requests](https://github.com/illinois/queue/pulls) made and [4 new Issues](https://github.com/illinois/queue/issues) logged for the project!
```
Comment (redsn0w422) [inline _posts/2018-04-24-illinihackers-queue-dev-night-recap.markdown]: Can you link to the specific PR's and issues that were made, in like a table?

PR name, person who made it (GitHub handle), and link to PR

**Pull request merged**